### ISSUE TITLE
jdbc_streaming version bundled in Logstash

### ISSUE THREAD

**Issue by cpiment** [elastic/logstash#11455]
Hi,

I'm trying to use the jdbc_streaming filter in Logstash 7.5.0 but I have found that the version bundled with Logstash 7.5 is 1.0.7, but in the documentation 

https://www.elastic.co/guide/en/logstash/current/plugins-filters-jdbc_streaming.html

The version for 7.5 is declared as 1.0.10.

I am running an issue where I want to use the prepared statements feature, but I can't because it was introduced in a later version.

- Version: Logstash 7.5.0
- Operating System: Linux (CentOS 7.6)
- Steps to Reproduce: Try to use the "use_prepared_statements" flag in jdbc_streaming configuration. Logstash won't start with this error:

`[2019-12-19T09:12:50,341][ERROR][logstash.filters.jdbcstreaming] Unknown setting 'use_prepared_statements' for jdbc_streaming`

Thanks for your help

**Comment by kares** [comment]
LS 7.5.1 and the plugin is [up-to-date there](https://github.com/elastic/logstash/pull/11393/files), also you could use `bin/logstash-plugin` helper to update

**Comment by cpiment** [comment]
Thanks. I didn't realize that LS 7.5.1 was released yesterday and that the plugin version was raised there.

I used the `logstash-plugin update` command to update the plugin locally, I raised the issue to warn the difference between the documentation of the filter plugin in the Logstash documentation and the plugin bundled with Logstash. But there is no such difference, because I wasn't using the latest LS version. My bad.
